lib/g_places.js

```diff
@@ -0,0 +1,83 @@
+https = require('https');
+
+/**
+ * function to search google places
+ * @param  {[object] name, lat, long from input form}
+ * @return {[object] gp - literal with google place values for newRest }
+ */
+module.exports = function(params, callback) {
+
+  var apikey = 'AIzaSyCY1BQZJ8B4_QjNr-UDFGvtInFs9SF7JM0';
```

Comment:
Agree

On Friday, July 17, 2015, Andrew <notifications@github.com> wrote:

> In lib/g_places.js
> <https://github.com/andyjac/hinton/pull/134#discussion_r34944018>:
>
> > @@ -0,0 +1,83 @@
> > +https = require('https');
> > +
> > +/**
> > + * function to search google places
> > + * @param  {[object] name, lat, long from input form}
> > + * @return {[object] gp - literal with google place values for newRest }
> > + */
> > +module.exports = function(params, callback) {
> > +
> > +  var apikey = 'AIzaSyCY1BQZJ8B4_QjNr-UDFGvtInFs9SF7JM0';
>
> i mostly just meant the code. probably not a huge deal if someone sees our
> api key for this, but still a good practice to keep it private i think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andyjac/hinton/pull/134/files#r34944018>.
>


-- 
Bob Day
Rialto Advisors
(206) 323-9664
LinkedIn <http://linkedin.com/in/daybob>